Makefile

```diff
@@ -9,9 +9,6 @@
 # Apply the contents of the .env to the terminal, so that the docker-compose file can use them in its builds
 export $(shell sed 's/=.*//' .env)
 
-.DEFAULT : help
-.PHONY : setup close clean build-backend run-backend build-web run-web database app api db-setup db-migrate db-rollback n8n-setup n8n-export clamav install test cypress lint lint-fix format format-fix help
```

Comment:
Make doesn't care if recipes are "phony" or not?

Reply:
nothing changed when I took it out, and it was already missing things, so I just removed it